packages/forms-web-app/src/controllers/projects/documents/utils/dates/setters/set-dates-filter-form-group-error-message-text.js

```diff
@@ -0,0 +1,5 @@
+const setDatesFilterFormGroupErrorMessageText = (formGroup, errorMessageText) => {
```

Comment:
this is mutating by reference. A better approach would be to return the object wit the key changed.